SwedbankPaySDK/Classes/SwedbankPaySDKViewModel.swift

```diff
@@ -49,18 +54,18 @@ final class SwedbankPaySDKViewModel: NSObject {
         
         /// If `configuration.pinCertificates` is not empty, pin certificates found in Bundle
         if let pinPublicKeys = self.configuration?.pinPublicKeys, !pinPublicKeys.isEmpty {
-            var serverTrustPolicies: [String : ServerTrustPolicy] = [:]
+            var pinEvaluators: [String : PublicKeysTrustEvaluator] = [:]
```

Comment:
Codacy found an issue: [Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-ios/pullRequest?prid=6081366)